lib/Mir.ml

```diff
@@ -50,11 +54,12 @@ type unsizedtype = Ast.unsizedtype [@@deriving sexp, hash]
 type constraint_check =
   {ccfunname: string; ccvid: string; cctype: sizedtype; ccargs: expr list}
 
+and formal_params = (adtype * string * unsizedtype) list
```

Comment:
Could you give `formal_params` a more informative name? E.g. `fun_arg_decl`.

Reply:
that's literally what a formal parameter is, haha. https://chortle.ccsu.edu/java5/Notes/chap34A/ch34A_3.html

Reply:
actually I guess those are just for identifiers, technically. I'll change it to fun_arg_decl, which has no pre-existing meaning and might be easier for people to deal with.

Reply:
Ah, right! I'd just never heard that term.